lib/tasks/ont_data.rake

```diff
@@ -121,30 +123,33 @@ namespace :ont_data do
         break if safe_peek(pool_enum).blank?
 
         position = position_cycle.next
-        flowcell_id = format('F%05d', position)
-        Ont::Flowcell.new(flowcell_id:, position:, run:, pool: pool_enum.next)
+        pool = pool_enum.next
+        flowcell_id = format('F%05d', flowcell_id_enum.next)
```

Comment:
What does this line do? I have some validation in the front end to ensure the Flowcell ID has: 3 letters followed by at least 3 numbers. 

I think this is what Alex specified, that a flowcell looks something like `ABC123`

Reply:
Also noticed the flowcell validation `F%05d` is specified in a few places, maybe that could get pulled out somewhere?

Reply:
I have defined Ont::Flowcell::FLOWCELL_ID_FORMAT = 'ABC%03d' and used it in other places. It is used in factories and ont_data:create task.